### PR TITLE
Update Hero Icon Placement - pin to the right

### PR DESCRIPTION
**Pull request by hqhhuang**
Bug/issue #, if applicable: 89970511

## Summary
With this PR, we will only have 1 icon pinned to the right in the hero section.
I also removed the white space in the SVG of the 4 icons used in the hero section to comply with the design.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests - NA
- [x] Ran `npm test`, and it succeeded
- [ ] Updated documentation if necessary - NA

### CODE REVIEW

**Comment by biamx3** [comment]
Looks good to me! Approved.

**Comment by hqhhuang** [comment]
@swift-ci test

**Comment by hqhhuang** [comment]
@swift-ci test

File: src/components/DocumentationTopic/DocumentationHero.vue [hunk 1]
```diff
@@ -100,40 +98,36 @@ $doc-hero-icon-color: dark-color(fill-secondary) !default;
     top: 0;
   }
 
+  .icon {
+    position: absolute;
+    margin-top: 10px;
+    margin-right: $doc-hero-icon-spacing;
```
Comment (dobromir-hristov) [inline src/components/DocumentationTopic/DocumentationHero.vue:104]: Is the little offset on `top and `right`, intentional?

image

**Review by dobromir-hristov — APPROVED** [review]
Looks good. Left a small comment in there.

**Comment by hqhhuang** [comment]
@swift-ci test

**Comment by hqhhuang** [comment]
@swift-ci test

**Pull request merged**